packages/core-components/src/components/DependencyGraph/DependencyGraph.tsx

```diff
@@ -262,7 +262,7 @@ export function DependencyGraph({
     <svg
       ref={containerRef}
       {...svgProps}
-      width={maxWidth}
+      width="100%"
       height={maxHeight}
```

Comment:
Probably makes sense to set height to `100%` too?

Reply:
I tested with height 100% as well, that doesn't look good.